examples/module/controller_parameters_tuner/commons.py

```diff
@@ -0,0 +1,88 @@
+import torch
+
```

Comment:
Like I said before, we can use Lietensor's built in functionality for these common operation.